### PR TITLE
Prepare changelog readme for revision 4017

### PR DESCRIPTION
**Pull request by frankeros**
Hi team, 
this update the changelog and readme for revision 4017
Feel free to update the titles for easier reading

### CODE REVIEW

**Review by pablomarga — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,44 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
+## Wazuh v4.0.4 - Kibana 7.10.0 , 7.10.2 - Revision 4017
```
Comment (Desvelao) [inline CHANGELOG.md:5]: It should be the `Revision 4016`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,44 @@
...
+- Fix discovery navigation [#2845](https://github.com/wazuh/wazuh-kibana-app/issues/2845)
+- Search file editor gone [#2843](https://github.com/wazuh/wazuh-kibana-app/issues/2843)
+- Fix Agent Search Bar - Regex Query Interpreter [#2834](https://github.com/wazuh/wazuh-kibana-app/issues/2834)
+- Fixed accordion style breaking [#2833](https://github.com/wazuh/wazuh-kibana-app/issues/2833)
+- Fix metrics are not updated after a bad request in search input [#2830](https://github.com/wazuh/wazuh-kibana-app/issues/2830)
+- Solaris should not show vulnerabilities module [#2829](https://github.com/wazuh/wazuh-kibana-app/issues/2829)
+- Update agents' info in Management Status after changing cluster node selected [#2828](https://github.com/wazuh/wazuh-kibana-app/issues/2828)
+- Fix mitre framework tab crash [#2821](https://github.com/wazuh/wazuh-kibana-app/issues/2821)
+- Changed ping request to default request. Added delay and while to che… [#2820](https://github.com/wazuh/wazuh-kibana-app/issues/2820)
+- Removed kibana alert for security [#2806](https://github.com/wazuh/wazuh-kibana-app/issues/2806)
```
Comment (Desvelao) [inline CHANGELOG.md:25]: Why are these changes in the `Adapt for Kibana 7.10.0` block and not in `Fixed` or `Changed`?

Comment (frankeros) [reply CHANGELOG.md:25]: Because they are changes introduced in the adaptation, they don't come from a previous version.

**Review by gabiwassan — APPROVED** [review]
LGTM

**Pull request merged**